deploy/test-environments/cis/main.tf

```diff
@@ -1,5 +1,6 @@
 provider "aws" {
-  region = var.region
+  region      = var.region
+  max_retries = 30
```

Comment:
why is this needed? 30 seems kind of a lot

Reply:
The default value is [25](https://registry.terraform.io/providers/hashicorp/aws/4.15.0/docs#max_retries-3), but we still encountered an API error when creating the EKS cluster. So, I explicitly increased this value to 30.